src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -314,8 +323,17 @@ export class ESLToggleable extends ESLBaseElement {
 
   /** Actions to execute on show request */
   @bind
-  protected _onShowRequest(): void {
-    this.show();
+  protected _onShowRequest(e: CustomEvent<ESLShowRequestDetails>): void {
+    const target = e.target;
+    const detail = e.detail;
+    if (!target) return;
+    let showOptions;
+    if (detail) {
+      showOptions = Object.assign({delay: detail.delay} || {}, {...detail.params} || {});
```

Comment:
It's better to send exact showDelay in that case.

Reply:
+ Why do we need to create a copy here `{...detail.params}` why can't we use it as it is.
`Object.assign` doesn't modify anything except first (target) object